src/main/target/IFLIGHT_F745_AIO_V2/target.h

```diff
@@ -0,0 +1,188 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define TARGET_BOARD_IDENTIFIER "IFRC"
+
+#define USBD_PRODUCT_STRING "IFLIGHT_F745_AIO_V2"
+
+#define ENABLE_DSHOT_DMAR       true
+
+#define LED0_PIN   PC13
+
+#define USE_BEEPER
+#define BEEPER_PIN PD2
+#define BEEPER_INVERTED
+
+//**********Gyro Acc *************//
+
+#define USE_ACC
+#define USE_GYRO
+#define USE_DUAL_GYRO
+
+#define USE_SPI_GYRO
+//#define USE_ACCGYRO_BMI270
+#define USE_GYRO_SPI_MPU6000
+#define USE_ACC_SPI_MPU6000
+
+#define USE_EXTI
+#define USE_GYRO_EXTI
```

Comment:
we may have issue with this line (`USE_GYRO_EXTI`) only in BMI dev-branches until problem resolved.